src/components/ProfileView/ProfileCourseList.tsx

```diff
@@ -0,0 +1,140 @@
+import { useTheme } from '@mui/material/styles';
+import { Box, Tooltip, Typography } from '@mui/material';
+import { Chip } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import IconButton from '@mui/material/IconButton';
+import TimerIcon from '@mui/icons-material/Timer';
+import { useState } from 'react';
+import { formatDate, daysUntilStart } from '@/lib/timedateutils';
+
+export interface ProfileCourseListProps {
+  headerText: string;
+  courses: Course[];
+  open: boolean;
+  timer?: boolean;
+}
+
+export default function ProfileCourseList({
+  headerText,
+  courses,
+  open,
+  timer,
+}: ProfileCourseListProps) {
+  const { palette } = useTheme();
+  const [isCollapsed, setIsCollapsed] = useState(open);
+
+  const handleToggleCollapse = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  return (
+    <Box
+      sx={{
+        paddingTop: '10px',
+      }}
+    >
+      <Typography
+        sx={{
+          backgroundColor: palette.darkBlue.main,
+          color: palette.white.main,
+          paddingLeft: '10px',
+        }}
+        variant="subtitle2"
+        data-testid="listHeader"
+      >
+        {`${headerText} (${courses.length})`}
+        <Tooltip
+          title={isCollapsed ? 'Close' : 'Expand'}
+          arrow
+          placement="right"
+        >
+          <IconButton
+            sx={{ color: palette.white.main }}
+            onClick={handleToggleCollapse}
+            data-testid="listControls"
+          >
+            {isCollapsed ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+          </IconButton>
+        </Tooltip>
+      </Typography>
+      {!isCollapsed ? null : (
+        <>
+          {courses.length === 0 ? (
+            <Typography
+              sx={{
+                padding: '10px',
+              }}
+              variant="body2"
+            >
+              No courses to show.
+            </Typography>
+          ) : (
+            <List
+              style={{
+                backgroundColor: palette.secondary.main,
+              }}
+            >
+              {courses.map((course: Course, count: number) => (
+                <React.Fragment key={course.id}>
+                  <Link
+                    href={`/profile/?courseId=${course.id}`}
+                    color="inherit"
+                    style={{ textDecoration: 'none' }}
+                  >
+                    <ListItem
+                      key={course.id}
+                      style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                      }}
+                      onMouseEnter={(e) =>
+                        (e.currentTarget.style.backgroundColor =
+                          palette.secondary.light)
+                      }
+                      onMouseLeave={(e) =>
+                        (e.currentTarget.style.backgroundColor = 'transparent')
```

Comment:
This effect would also be achievable using the hover selector: 
```
 <ListItem
            key={course.id}
             sx={{
                  display: 'flex',
                  justifyContent: 'space-between',
                  backgroundColor: 'transparent',
                  '&:hover': {
                       backgroundColor: palette.secondary.light,
                    },
                  }}
                 }>
```

Reply:
That is probably how others would do it too, I'll change to that so it's more unifrom across the codebase.